src/Microsoft.Identity.Web.Certificateless/ManagedIdentityClientAssertion.cs

```diff
@@ -34,6 +37,7 @@ public ManagedIdentityClientAssertion(string? managedIdentityClientId)
                     ExcludeVisualStudioCodeCredential = true,
                     ExcludeVisualStudioCredential = true
                 });
+            _tokenExchangeUrl = tokenExchangeUrl ?? CertificatelessConstants.DefaultTokenExchangeUrl;
```

Comment:
nit: tokenExchangeUrl is never null here because it's already set to the default value.

Reply:
I thought so too initially, but when I checked, "null" is a valid value to pass as the param and will override the default value.

Reply:
Closing this thread - pls reopen if you have more concerns 😄

Reply:
@kellyyangsong Indeed yes. Someone can explicitly pass a null, then default value doesn't get assigned (it only gets assigned when nothing is passed.)